packages/beacon-node/package.json

```diff
@@ -89,7 +89,7 @@
     "test:spec:bls": "vitest --run --config vitest.spec.config.ts --dir test/spec/bls/",
     "test:spec:general": "vitest --run --config vitest.spec.config.ts --dir test/spec/general/",
     "test:spec:minimal": "LODESTAR_PRESET=minimal vitest --run --config vitest.spec.config.ts --dir test/spec/presets/",
-    "test:spec:mainnet": "LODESTAR_PRESET=mainnet vitest --run --config vitest.spec.config.ts --dir test/spec/presets/",
+    "test:spec:mainnet": "vitest --run --config vitest.spec.config.ts --dir test/spec/presets/",
```

Comment:
I don't get why `LODESTAR_PRESET=mainnet` is removed here, if these are specific to mainnet preset and I would expect them to run against that preset no matter what values i have in my .env files.

Same goes for any tests that only pass for a specific preset, we should not allow to override it there

Reply:
Because `LODESTAR_PRESET=mainnet ` is passed as default form the `.env.test` file.